model/host/host.go

```diff
@@ -768,11 +767,11 @@ func CountInactiveHostsByProvider() ([]InactiveHostCounts, error) {
 	return counts, nil
 }
 
-// FindAllRunningContainers finds all running containers
+// FindAllRunningContainers finds all the containers that are currently running
 func FindAllRunningContainers() ([]Host, error) {
```

Comment:
This should take in distro as a parameter. You don't want all containers, just the ones relevant to the distro you're planning

Reply:
Currently I'm using this just to find how many existing containers are up so I can see if there is any room. Why is it relevant if a container is related to a distro in this case?

Reply:
This was from before we discussed the "how do we specify that certain hosts can only have certain types of containers" where I assumed it would be stored in the distro. This is fine as is but it'll change after whatever we do to answer this question